templates/manifest.json

```diff
@@ -1,5 +1,39 @@
 {
     "templates": [
+        {
+            "id": "python-empty",
+            "name": "python-empty",
+            "label": "Empty Python project",
+            "category": "python",     
+            "description": "Empty starter Actor. A blank slate with minimal starting code. Useful for quick scripting without having to delete bunch of stuff from other templates.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/python-empty.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 1024,
+                "timeoutSecs": 3600
+            },
+            "showcaseFiles": [
+                "src/main.py",
+                "src/__main__.py"
+            ]
+        },
+        {
+            "id": "python-start-scrape-single-page",
+            "name": "python-start-scrape-single-page",
+            "label": "Scrape single page",
+            "category": "python",
+            "description": "A simple Python example of core Actor and Apify SDK features. It reads and validates user input with schema, computes a result and saves it to storage.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/python-start-scrape-single-page.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 1024,
+                "timeoutSecs": 3600
+            },
+            "showcaseFiles": [
+                "src/main.py",
+                "src/__main__.py"
+            ]
+        },
```

Comment:
These should probably not be the first Python templates, but rather the last ones, same as how @jbartadev put the `js-empty` template as last of the JS templates in https://github.com/apify/actor-templates/pull/148.